client/src/containers/section/map/index.tsx

```diff
@@ -144,7 +147,8 @@ export default function MapContainer() {
         bounds={tmpBounds}
         minZoom={minZoom}
         maxZoom={maxZoom}
-        mapStyle={'https://demotiles.maplibre.org/style.json'}
+        // @ts-expect-error TODO: Map style is not matching the type
+        mapStyle={mapStyle}
```

Comment:
You can cast the variable:
```suggestion
        mapStyle={mapStyle as MapboxStyle}
```

`MapboxStyle` comes from `react-map-gl`.